src/test/resources/anet.graphql

```diff
@@ -1359,4 +1362,24 @@ input UserActivitySearchQueryInput {
 enum UserActivitySearchSortBy {
   COUNT
   NONE
-}
\ No newline at end of file
+}
+
+""""""
+input EntityAvatar {
```

Comment:
**Always** update this file using the proper command ( `yarn run generate-graphql-schema` ) instead of manually…